core/config/envvar/schema.go

```diff
@@ -171,6 +171,7 @@ type ConfigSchema struct {
 	EvmGasBumpWei         *big.Int `env:"ETH_GAS_BUMP_WEI"`
 	EvmGasFeeCapDefault   *big.Int `env:"EVM_GAS_FEE_CAP_DEFAULT"`
 	EvmGasLimitDefault    uint32   `env:"ETH_GAS_LIMIT_DEFAULT"`
+	EvmGasLimitMax        uint32   `env:"ETH_GAS_LIMIT_MAX"`
```

Comment:
How do we decide if this var should be ETH_* or EVM_*?
This seems more appropriate for EVM_* i think.

Reply:
IIRC `EVM_` is legacy and `ETH_` is newer/preferred barring any other reasons, but I generally try to follow local style. In this case, there are already three other closely related gas limit params with `ETH_GAS_LIMIT_*`.

But regardless, these will go away soon with the new config.